react/components/MiniCartFooter.js

```diff
@@ -0,0 +1,119 @@
+import React, { PureComponent, Fragment } from 'react'
+import classNames from 'classnames'
+import ProductPrice from 'vtex.store-components/ProductPrice'
+import { Button, Spinner } from 'vtex.styleguide'
+import { FormattedMessage } from 'react-intl'
+import PropTypes from 'prop-types'
+
+import { MiniCartPropTypes } from '../propTypes'
+import minicart from '../minicart.css'
+
+class MiniCartFooter extends PureComponent {
+  static propTypes = {
+    shippingCost: PropTypes.number,
+    large: PropTypes.bool,
+    isUpdating: PropTypes.bool,
+    totalValue: PropTypes.number.isRequired,
+    buttonLabel: PropTypes.string.isRequired,
+    showDiscount: MiniCartPropTypes.showDiscount,
+    discount: PropTypes.number,
+    labelDiscount: PropTypes.string,
+    showShippingCost: MiniCartPropTypes.showShippingCost,
+  }
+
+  handleClickButton = () => location.assign('/checkout/#/cart')
+
+  render() {
+    const { 
+      shippingCost, 
+      large, 
+      isUpdating, 
+      totalValue, 
+      buttonLabel, 
+      showDiscount, 
+      discount, 
+      labelDiscount,
+      showShippingCost
+     } = this.props
+
+    const priceAndDiscountClasses = classNames(
+      `${minicart.contentDiscount} w-100 flex justify-end items-center mb3`,
+      {
+        'pv3': large,
+      }
+    )
+
+    const checkoutButtonClasses = classNames(
+      '',
+      {
+        'bb bw4 bw2-m b--transparent': large,
+      }
+    )
+
+    const shouldShowShippingCost = showShippingCost && shippingCost > 0
```

Comment:
If the `shippingCost` is 0, shouldn't it be showed? Or maybe a `Free Shipping` msg?

Reply:
Pensei nisso de mostrar uma string "Gratis" caso fosse 0, porém cheguei a conclusão que isso não deve ocorrer (eu acho). Se o custo for zero, o item não deve entrar em orderForm.totalizers (eu acho), pois ele não ta agregando nada ao custo do orderForm.value